src/conic.jl

```diff
@@ -116,7 +116,7 @@ function MathProgBase.loadproblem!(
     m.is_conic_solver = (applicable(MathProgBase.ConicModel, m.cont_solver) && m.cont_solver != MathProgBase.defaultNLPsolver)
 
     # Keep default soc disaggregate only if conic solver
-    m.disaggregate_soc = (m.is_conic_solver && m.disaggregate_soc == :default) ? Symbol("true") : Symbol("false")
+    m.disaggregate_soc = (m.is_conic_solver && m.disaggregate_soc == _disagg_soc_default) ? _disagg_soc_on : m.disaggregate_soc
```

Comment:
What happens when we get an NLP solver? Status is left as ``_disagg_soc_default`` instead of ``_disagg_soc_off``?